src/modules/person/api/personen-query.param.ts

```diff
@@ -33,7 +33,7 @@ export class PersonenQueryParams extends PagedQueryParams {
     })
     public readonly vorname?: string;
 
-    @AutoMap(() => String)
```

Comment:
Could remove all `@AutoMap` from this file, PersonenQueryParams is no longer used in any mappings